cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -155,3 +178,62 @@ def test_tenant_quota_enforce_via_service_cloud(request, appliance, provider, se
     def delete():
         provision_request.remove_request()
         catalog_item.delete()
+
+
+# Args of parametrize is the list of navigation parameters to order catalog item.
+@pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
```

Comment:
Formatting a bit off here, keep the comment block together and the parametrize lines together, instead of this splitting the comment block please.